replica/database.hh

```diff
@@ -1469,6 +1472,9 @@ private:
     reader_concurrency_semaphore _compaction_concurrency_sem;
     reader_concurrency_semaphore _system_read_concurrency_sem;
 
+    // The view update read concurrency semaphore is not selected based on the scheduling group
+    // and is used for all view update reads.
+    reader_concurrency_semaphore _view_update_read_concurrency_sem;
```

Comment:
This is probably fine for open source, but in enterprise user writes can be scheduled on different scheduling groups, depending on the service level used (correct me if I'm wrong @Jadw1).

Reply:
From what I've saw we'll probably need to use `reader_concurrency_semaphore_group` for this in enterprise so I'll need to fix this when backporting to enterprise